react-frontend/src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import openMainFrameLogo from "../../images/openmainframeproject-color-1.png";
+
+const Footer = () => {
+  return (
+    <footer className="text-white p-6 bg-gradient-to-r from-blue-900 to-black">

```

Comment:
This gradient looks appealing but "Project" from Open Mainframe Project is almost invisible. Could we do something about it?
![image](https://github.com/openmainframeproject/software-discovery-tool/assets/120312681/be587041-33ef-4b99-801c-90476dff438d)

Reply:
Yes, I will change the background color. I need to find the right shade. If you have any suggestions, please let me know. A gradient would probably look good.